apps/web/src/pages/tx/[hash].tsx

```diff
@@ -230,9 +229,9 @@ const Tx: NextPage = () => {
   }
 
   const decodedData =
-    rawTxData && rawTxData.decodedFields
-      ? parseDecodedData(rawTxData.decodedFields)
-      : null;
+    rawTxData?.decodedFields?.type === "optimism"
```

Comment:
what are the benefits from this change? I liked the previous simplicity because it allowed us to have an agnostic decodedFields json object where we can add more data as we need it without defining any schema.

Reply:
This change comes from this [conversation](https://github.com/Blobscan/blobscan/pull/608#discussion_r1819899633)

We already need to parse the `decodedFields` object on the client, and it must comply with a specific type to render the ‘Decoded Fields’ section. Why not handle this directly on the API, so the client isn’t responsible for parsing and the interaction with our API is simplified (via web or our rest api)?